src/components/NewHomePage/LoanCards.vue

```diff
@@ -0,0 +1,407 @@
+			return this.loan?.distributionModel || '';
+		},
+		imageHash() {
+			return this.loan?.image?.hash ?? '';
+		},
+		loanUse() {
+			if (this.loan?.distributionModel === 'direct') {
+				return `Loan for ${this.loan?.use}`;
+			}
+			return `Loan ${this.loan?.use}`;
+		},
+		fundraisingPercent() {
+			if (this.loan?.distributionModel) {
+				return this.loan?.fundraisingPercent * 100;
+			}
+			return 0;
+		},
+		unreservedAmount() {
+			return this.loan?.unreservedAmount ?? '0';
+		},
+		timeLeft() {
+			return this.loan?.fundraisingTimeLeft ?? '';
+		},
+		formattedLocation() {
+			if (this.distributionModel === 'direct') {
+				const formattedString = `${this.city}, ${this.state}, ${this.countryName}`;
+				return formattedString;
+			}
+			if (this.countryName === 'Puerto Rico') {
+				const formattedString = `${this.city}, PR`;
+				return formattedString;
+			}
+			return this.countryName;
+		},
+		allSharesReserved() {
+			if (parseFloat(this.loan?.unreservedAmount) === 0) {
+				return true;
+			}
+			return false;
+		},
+		inBorrowerProfilePage() {
+			return this.$route.path.includes('funded');
+		},
+		loanAmount() {
+			return this.loan?.loanAmount;
+		},
+	},
+	methods: {
+		createViewportObserver() {
+			// Watch for this element being in the viewport
+			this.viewportObserver = createIntersectionObserver({
+				targets: [this.$el],
+				callback: entries => {
+					entries.forEach(entry => {
+						if (entry.target === this.$el && entry.intersectionRatio > 0) {
+							// This element is in the viewport, so load the data.
+							this.loadData();
+						}
+					});
+				}
+			});
+			if (!this.viewportObserver) {
+				// Observer was not created, so call loadData right away as a fallback.
+				this.loadData();
+			}
+		},
+		destroyViewportObserver() {
+			if (this.viewportObserver) {
+				this.viewportObserver.disconnect();
+			}
+		},
+		loadData() {
+			if (!this.queryObserver) {
+				this.queryObserver = watchLoanCardData({
+					apollo: this.apollo,
+					loanId: this.loanId,
+					loanCardQuery,
+					callback: result => this.processQueryResult(result),
+				});
+			}
+		},
+		processQueryResult(result) {
+			if (result.error) {
+				console.error(result.error);
+				this.$showTipMsg('There was a problem loading your loan recommendations', 'error');
+				try {
+					Sentry.withScope(scope => {
+						scope.setTag('wizard_stage', 'results');
+						scope.setTag('loan_id', this.loanId);
+						Sentry.captureException(result.error);
+					});
+				} catch (e) {
+					// no-op
+				}
+			}
+
+			this.isLoading = false;
+			this.loan = result.data?.lend?.loan || null;
+		},
+	},
+	mounted() {
+		this.createViewportObserver();
+	},
+	beforeDestroy() {
+		this.destroyViewportObserver();
+	},
+	watch: {
+		// When loan id changes, update watch query variables
+		loanId(loanId) {
+			if (this.queryObserver) {
+				this.queryObserver.setVariables({
+					loanId,
+				});
+			}
+		},
+	},
+};
+</script>
+<style lang="scss" scoped>
```

Comment:
Is this a temporary experiment or new permanent element? If it's permanent, consider changing to tailwind + postcss. I see this uses the old foundation import as well.

Reply:
We should not be importing `settings` or `foundation` any longer.

Please refactor this to use `tw-` classes for the colors you need, which you can apply within the `postcss` setup if you'd like. You can keep some of this raw css but most of it should be converted to `tw-` classes except where tailwindcss doesn't support it. _I think everything below is supported._

Reply:
Checkout [TheHeader](https://github.com/kiva/ui/blob/main/src/components/WwwFrame/TheHeader.vue) and https://github.com/kiva/ui/blob/main/src/components/BorrowerProfile/WhatIsKivaModal.vue#L136 for samples of the new style usage paradigm.